Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx

```diff
@@ -314,31 +314,31 @@
     <comment>Text for Never</comment>
   </data>
   <data name="NewEvent" xml:space="preserve">
-    <value>Nouvel évènement</value>
+    <value>Nouvel événement</value>
     <comment>Title for Editor UI when create new appointment</comment>
   </data>
   <data name="NoDateSelected" xml:space="preserve">
     <value>Aucune date sélectionnée</value>
     <comment>Text for No Date Selected</comment>
   </data>
   <data name="NoEvents" xml:space="preserve">
-    <value>Pas d'événements</value>
+    <value>Aucun événement</value>
     <comment>Text for No Events</comment>
   </data>
   <data name="November" xml:space="preserve">
-    <value>novembre</value>
+    <value>Novembre</value>
     <comment>Text for November</comment>
   </data>
   <data name="October" xml:space="preserve">
-    <value>octobre</value>
+    <value>Octobre</value>
     <comment>Text for October</comment>
   </data>
   <data name="Ok" xml:space="preserve">
-    <value>D'accord</value>
+    <value>Ok</value>
     <comment>Content for Ok button that is displayed when edit/delete recurring appointment</comment>
   </data>
   <data name="OutRangeOccurrencedEditingMessageAlert" xml:space="preserve">
-    <value>Impossible de replanifier une occurrence du rendez-vous récurrent [] s'il ignore une occurrence ultérieure du même rendez-vous.</value>
+    <value>Alerte de message d'édition d'occurrence hors plage</value>
```

Comment:
Check

Reply:
Changed text to **Alerte de message : occurrence hors de la plage**